Helper/ShellContextMenu.cs

```diff
@@ -0,0 +1,1584 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Runtime.InteropServices;
+using System.Drawing;
+using System.Windows.Forms;
+using System.IO;
+using System.Security.Permissions;
+
+namespace Peter
+{
+    /// <summary>
```

Comment:
Just adding a comment to show where the code is from.
```suggestion
    //  Code from https://www.codeproject.com/Articles/22012/Explorer-Shell-Context-Menu:
    /// <summary>
```